R/utils.R

```diff
@@ -121,6 +121,9 @@ ask <- function(question, default = FALSE) {
   if (!interactive())
     return(default)
 
+  # be verbose in this scope, as we're asking the user for input
+  renv_scope_options(renv.verbose = TRUE)
```

Comment:
That only affects the single `writef()` below, right?

Reply:
It looks like there's 3 writef's below, although two are just there to print a newline. But yeah, that's right.